integration/mediation-tests/tests-transport/src/test/resources/artifacts/ESB/ssl/axis2.xml

```diff
@@ -105,7 +101,7 @@
 
     <!-- This deployment interceptor will be called whenever before a module is initialized or -->
     <!-- service is deployed -->
-    <listener class="org.wso2.carbon.core.deployment.DeploymentInterceptor"/>
+    <!--listener class="org.wso2.carbon.core.deployment.DeploymentInterceptor"/-->
```

Comment:
Shall we remove this instead of commenting out?

Reply:
Done in #575